daxa/process/erosita/clean.py

```diff
@@ -0,0 +1,322 @@
+
+    # Converting to the right unit                              
+    else:
+        source_size = source_size.to('arcsec')
+
+    # Checking user's choice for the binsize parameter
+    if not isinstance(binsize, int):
+        raise TypeError("The binsize argument must be an integer.")
+
+    # Checking the validity of the binsize value
+    elif binsize <= 0:
+        raise ValueError("The binsize argument may not be negative or equal to 0.")
+    
+    # Checking user's choice for the detml parameter
+    if not isinstance(detml, (int, float)):
+        raise TypeError("The detml argument must be an integer or a float.")
+    
+    # Checking user's choice for the source_like parameter
+    if not isinstance(source_like, (int, float)):
+        raise TypeError("The source_like argument must be an integer or a float.")
+    
+    # Checking user's choice for the threshold parameter
+    if not isinstance(threshold, Quantity):
+        raise TypeError("The threshold argument must be an astropy quantity in units that can "
+                        "be converted into counts/deg^2/s.")
+
+    # Checking it is in the correct units
+    elif not threshold.unit.is_equivalent('sb_rate'):
+        raise UnitConversionError("The threshold argument must be an astropy quantity in units that can "
+                                  "be converted into counts/deg^2/s.")
+    
+    # Converting to the right unit                              
+    else:
+        threshold = threshold.to('sb_rate')
+    
+    # Checking user's choice for the max_threshold parameter
+    if not isinstance(max_threshold, Quantity):
+        raise TypeError("The max_threshold argument must be an astropy quantity in units that can "
+                        "be converted into counts/deg^2/s.")
+
+    # Checking it is in the correct units
+    elif not max_threshold.unit.is_equivalent('sb_rate'):
+        raise UnitConversionError("The max_threshold argument must be an astropy quantity in units that can "
+                                  "be converted into counts/deg^2/s.")
+    
+    # Converting to the right unit                              
+    else:
+        max_threshold = max_threshold.to('sb_rate')
+    
+    # Checking user's choice for the mask_iter parameter
+    if not isinstance(mask_iter, int):
+        raise TypeError("The mask_iter argument must be an integer.")
+
+    # Converting parameters from astropy units into a type the command line will accept
+    pimin = int(pimin.value)
+    pimax = int(pimax.value)
+    mask_pimin = int(mask_pimin.value)
+    mask_pimax = int(mask_pimax.value)
+    timebin = float(timebin.value)
+    source_size = float(source_size.value)
+    threshold = threshold.value
+    max_threshold = max_threshold.value
+
+    # These parameters we want DAXA to have control over, not the user
+    gridsize = 18   # Sections of the image a source detection is run over to determine a dynamic threshold
```

Comment:
FOV_radius is in arcminutes, and comes from the default in the flaregti documentation online. I definitely need to look into this parameter again because i'm unsure as to why you would want to change it/ what the effect of changing it would really be. 

Just opened a github issue now